db/schema.rb

```diff
@@ -1439,9 +1440,9 @@
 
   create_table "users_gdpr_delete_requests", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
-    t.string "email"
+    t.string "email", null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.integer "user_id"
+    t.integer "user_id", null: false
```

Comment:
I did not make this change

Reply:
@msarit I usually use `git add -p` to avoid committing changes I didn't mean to commit, _especially_ with `schema.rb`. This file is generated based on the current schema of your development DB, which may not _actually_ match production due to differences in how both were provisioned and various changes over time.

For example, production's current state was generated from running `rake db:migrate` every time, but the rest of us usually used something like `db:schema:load` or `db:setup` (which I believe uses `db:schema:load`) in development because running hundreds or thousands of migrations is slow and error-prone. If any PR with a migration was ever reverted without first running its associated `rake db:rollback` in production first, then `schema.rb` and the _actual_ schema will have diverged. It's one of my least favorite things about DB migrations. 🙂 

I just checked DEV's DB, though, and some of these `NOT NULL` constraints do exist there, so I'll probably put up a PR soon that puts all of us on the same page.